docs/develop/sdks-integrations/aws-secrets-manager.md

```diff
@@ -1,7 +1,6 @@
 ---
 sidebar_position: 3
 sidebar_label: AWS Secrets Manager
-pagination_next: null
```

Comment:
Typo?

Reply:
No. I need to remove this from this page as it will no longer be the last in the list. The new page is the last one in the list.